app/scripts/modules/core/src/pipeline/config/stages/awsCodeBuild/AwsCodeBuildStageForm.tsx

```diff
@@ -141,6 +141,20 @@ export function AwsCodeBuildStageForm(props: IFormikStageConfigInjectedProps) {
           />
         )}
       />
+      <FormikFormField
+        help={<HelpField id="pipeline.config.codebuild.secondarySourcesVersionOverride" />}
+        label="Secondary Sources Version"
+        name="secondarySourcesVersionOverride"
+        input={(inputProps: IFormInputProps) => (
+          <AwsCodeBuildSecondarySourcesVersionList
+            {...inputProps}
+            secondarySourcesVersionOverride={get(stage, 'secondarySourcesVersionOverride')}
+            updateSecondarySourcesVersion={(secondarySourcesVersionOverride: IAwsCodeBuildSecondarySourcesVersion[]) => onFieldChange('secondarySourcesVersionOverride', secondarySourcesVersionOverride)}
```

Comment:
If you refactor the `AwsCodeBuildSecondarySourcesVersionList` to be a Form Input then these two props can be removed